src/main/java/fi/dy/masa/tweakeroo/util/EntityRestriction.java

```diff
@@ -0,0 +1,37 @@
+package fi.dy.masa.tweakeroo.util;
+
+import fi.dy.masa.malilib.util.restrictions.UsageRestriction;
+import fi.dy.masa.tweakeroo.Tweakeroo;
+import net.minecraft.entity.EntityType;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+import java.util.List;
+import java.util.Set;
+
+public class EntityRestriction extends UsageRestriction<EntityType<?>>
+{
+    @Override
+    protected void setValuesForList(Set<EntityType<?>> set, List<String> names)
+    {
+        for (var name : names)
```

Comment:
I think I would write this as:
```java
            for (String name : names)
            {
                try
                {
                    Optional<EntityType<?>> type = Registry.ENTITY_TYPE.getOrEmpty(new Identifier(name));

                    if (type.isPresent())
                    {
                        set.add(type.get());
                        continue;
                    }
                }
                catch (Exception ignore) {}

                Tweakeroo.logger.warn("Invalid entity name in a black- or whitelist: '{}'", name);
            }
```

(Style notes: I don't like `var` to "hide" the type, and I use curly braces even for single line blocks.)